src/settings.py

```diff
@@ -0,0 +1,39 @@
+from pydantic import BaseModel
+import os
+import sys
+import logging
+
+
+class Settings(BaseModel):
+    """Class for keeping track of settings"""
+
+    guild_id: int
+    discord_token: str = os.environ["DISCORD_TOKEN"]
+    log_level: int = logging.INFO
+    logging_channel_id: int
+
+
+def prod() -> Settings:
+    return Settings(
+        guild_id=1172230157776466050,
+        logging_channel_id=int(os.environ["MOD_LOGGING_CHANNEL_ID"]),
```

Comment:
this should be a fixed channel ID as well. you can get this info from @clairefleuret